bpf/lib/lb.h

```diff
@@ -398,8 +398,12 @@ static __always_inline int reverse_map_l4_port(struct __ctx_buff *ctx, __u8 next
 				return ret;
 
 			if (port != old_port) {
+				/* This will change the SCTP checksum, which we cannot fix right now.
+				 * This will likely need kernel changes before we can drop this.
+				 */
+				if (nexthdr == IPPROTO_SCTP) return CTX_ACT_DROP;
```

Comment:
I think we should put the feature behind a flag rather than introduce runtime checks in the datapath code like this.

Reply:
I had requested explicit drop for this case to ensure that we don't end up just passing packets on with malformed checksums. Could we just wrap this in the feature flag check? It's kinda helpful to have an explicit mention here as well to highlight that this path is broken in the SCTP support.

Reply:
Yeah, I'm happy with this if it's behind a disabled-by-default feature flag.

Reply:
SGTM